### PR TITLE
QOL very up

### LINKED ISSUE #16: docker-compose QOL very up
- [ ] hot reload
  - [ ] frontend
  - [ ] message_broker
  - [ ] api_gateway

### PR DESCRIPTION
**Pull request by enokawa**
Closes: #16

## Description

QOL very up on local development.

## Details of this change

- [x] Hot-reload frontend
- [x] Hot-reload message_broker
- [x] Hot-reload api_gateway

## Information for Reviewer

Use [cosmtrek/air](https://github.com/cosmtrek/air) because the Air has many GitHub stars and ongoing maintained.

### CODE REVIEW

**Comment by enokawa** [comment]
@nari-z Please review.

File: api_gateway/.air.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[misc]
+clean_on_exit = true
```
Comment (nari-z) [inline api_gateway/.air.toml:2]: 👍

**Review by nari-z — COMMENTED** [review]
[IMO]
How about add to .gitignore
```
api_gateway/tmp/
message_broker/tmp/
```

**Comment by enokawa** [comment]
@nari-z No need to fix it because tmp directory will deleted after stopped Docker container. What do you think?

**Review by nari-z — APPROVED** [review]
LGTM

**Pull request merged**